modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first}.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing IPv6 subnet.
+* DNS is configured for the existing IPv6 subnet.
+* The IPv6 subnet is added to a {rh-openstack} router, and the router is configured to send router advertisements (RAs). 
+* The {rh-openstack} network accepts a minimum MTU size of 1380 bytes.
+* You created API and ingress virtual IP addresses (VIPs) as ports on the network and included those addresses in the `install-config.yaml` file.
+
```

Comment:
@MaysaMacedo Don't you think we should we mention that a mirror  with ipv6 interface is required to install?

Reply:
```suggestion
* You have a mirror registry with an IPv6 interface.
```